hbase-assembly/src/main/assembly/hadoop-three-compat.xml

```diff
@@ -115,6 +115,13 @@
               <exclude>junit:junit</exclude>
               <exclude>org.hamcrest:hamcrest-core</exclude>
               <exclude>org.mockito:mockito-core</exclude>
+              <!-- Exclude transitive dependencies of tomcat-jasper, not needed at runtime -->
```

Comment:
We do not need this for our release binaries? For client I think it is OK that we do not need to start a web server, but for hbase we do need to start master/rs webs?

Reply:
I tested out master startup in local mode. Was able to start with just tomcat-jasper, tomcat-jasper-el and tomcat-el-api and hence removed others. Also these 3 jars bring in all those classes which are getting removed due to removal of javax.el and javax.servlet.jsp.

Can keep these for safer side. Please let me know WDYT.

Reply:
Then we could just exclude them in the dependencyManagement or dependencies section in our pom file? We need to them in tests?

Reply:
Yes @Apache9, these are required at compile time to generate the jsp pages but i was able to load web pages w/o them during runtime. Hence excluding from assembly. 
Few others which were not needed at all, I have already excluded in  dependencies section for the dependency.

Reply:
Gentle ping @Apache9.

Reply:
Hi @Apache9 seems someone is interested in this patch. Could you please have another look at the changes and the above discussion and see if we are good here. Please let me know if otherwise.